### DISCUSSION TITLE
Ability to add comments to environments

### DISCUSSION THREAD

**Discussion by Jaakkonen** [Kong/insomnia#4554] [Ideas]
I have a use case with multiple ids being fixed in environments and it'd be nice to have ability to have `// ` prefixed lines for comments in the environment JSON (like the config JSON in VSCode). Currently such lines produce a syntax error but are highlighted as being comments, the best workaround I found was just having comments as variables with some prefix like `"_comment_id": "The id links to a thing"` but those aren't highlighted differently from normal fields.

**Comment by muuvmuuv** [comment]
Augh.. would be covert by this one here but its closed https://github.com/Kong/insomnia/issues/770
Coments would be a killer on large envs!

**Comment by spider4216** [comment]
Hi there! I agree with you, need comments feature too

**Comment by subnetmarco** [comment]
We would like to introduce comments on every file, including collections as well. We can start planning for this after summer.
